/**
 * Copyright (c) 2015 - The CM Authors <legal@clickmatcher.com>
 *   All Rights Reserved.
 *
 * This file is CONFIDENTIAL -- Distribution or duplication of this material or
 * the information contained herein is strictly forbidden unless prior written
 * permission is obtained.
 */
package cm;

message ItemBoostTermInfo {
  required string term = 5;
  required uint32 score = 6;
};

message ItemBoostRow {
  required uint32 num_impressions = 3;
  required uint32 num_clicks = 4;
  repeated ItemBoostTermInfo top_terms = 7;
}


